lib/linguist/languages.yml

```diff
@@ -3332,6 +3332,14 @@ NSIS:
   codemirror_mode: nsis
   codemirror_mime_type: text/x-nsis
   language_id: 242
+Nasl:
+  type: programming
+  extensions:
+  - ".nasl"
+  - ".inc"
```

Comment:
Could you add a search query in the original post for this extension as well?

`.inc` is already used by quite a few languages, so if we add it for Nasl as well, we're going to need a heuristic.

Reply:
Search query: Done!
Heuristic: Done! (see new commit)

Reply:
Please filter the search query using some NASL-specific keywords. The current query is returning all `.inc` files on GitHub.

Reply:
Added filtering using "Tenable" keyword, which narrows down the search a lot but at least returns no false-positives.